sdk/core/azure-core/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 - Fixed `Azure::DateTime::Parse()` validation if the result is going to exceed `9999-12-31T23:59:59.9999999` due to time zone, leap second, or fractional digits rounding up adjustments.
 
+- [[#3224]](https://github.com/Azure/azure-sdk-for-cpp/issues/3224) Fixed intermittent crash on macOS when logging is turned on.
```

Comment:
Missed removing that extra new line between the list of changes within the changelog:
```suggestion
- [[#3224]](https://github.com/Azure/azure-sdk-for-cpp/issues/3224) Fixed intermittent crash on macOS when logging is turned on.
```

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-cpp/pull/3255